mineinabyss-components/src/main/kotlin/com/mineinabyss/components/armorstandlock/LockArmorStand.kt

```diff
@@ -0,0 +1,18 @@
+package com.mineinabyss.components.armorstandlock
+
+import com.mineinabyss.idofront.serialization.UUIDSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.util.*
+
+@Serializable
+@SerialName("mineinabyss:lockable")
+data class LockArmorStand(
+    val owner: @Serializable(with = UUIDSerializer::class) UUID,
+    var lockState: Boolean,
+    val allowedAccess: MutableSet<@Serializable(with = UUIDSerializer::class) UUID?>
```

Comment:
Why is UUID nullable, it should either be in the list or not be in the list?

Reply:
True, set is never empty as owner will always be there